pkg/action/actioncontroller.go

```diff
@@ -149,3 +190,24 @@ func (ac *ActionController) createCRD() error {
 	}
 	return k8sutils.CreateCRD(resource, validateCRDInterval, validateCRDTimeout)
 }
+
+// getLatestMigrationStatus returns the a migrationschedule's latest migration's policy type and status
+func getLatestMigrationPolicyAndStatus(migrationSchedule storkv1.MigrationSchedule) (storkv1.SchedulePolicyType, *storkv1.ScheduledMigrationStatus) {
+	var lastMigrationStatus *storkv1.ScheduledMigrationStatus
+	var schedulePolicyType storkv1.SchedulePolicyType
+	for schedulePolicyType, policyMigration := range migrationSchedule.Status.Items {
+		if len(policyMigration) > 0 {
+			lastMigrationStatus = policyMigration[len(policyMigration)-1]
+			return schedulePolicyType, lastMigrationStatus
+		}
+	}
+	return schedulePolicyType, lastMigrationStatus
+}
+
+func isMigrationComplete(status storkv1.MigrationStatusType) bool {
```

Comment:
I feel we can add status == storkv1.MigrationStatusInitial as well in the condition here 
So new condition should be 
```
if status == storkv1.MigrationStatusInitial ||
                status == storkv1.MigrationStatusPending ||
		status == storkv1.MigrationStatusInProgress {
		return false
	}
```

Reply:
Done